qemu.go

```diff
@@ -66,6 +67,19 @@ const (
 
 const defaultQemuPath = "/usr/bin/qemu-system-x86_64"
 
+const (
+	defaultCPUs    = 2
+	defaultCores   = 1
+	defaultSockets = 2
+	defaultThreads = 1
+)
+
```

Comment:
So our internal benchmarks showed that we really want 1 socket only. Also the default CPUs should map the host cpu count and not a hard coded number. Basically we want the qemu smp command line to look like:
```
"-smp %d,sockets=1,cores=%d,threads=1", vCPUs"
```

And vCPUs is either the passed CPUs resource or the host cpu count if the former is 0.